sdcm/sct_events.py

```diff
@@ -339,6 +349,25 @@ def eval_filter(self, event):
         return result
 
 
+class EventsFilter(BaseFilter):
+    def __init__(self, event_class=None, regex=None):
```

Comment:
using strings for `event_class` is error-prone, use real event classes

Reply:
done